### DISCUSSION TITLE
button to show/hide Celestial Sphere

### DISCUSSION THREAD

**Discussion by nowandnever46** [Stellarium/stellarium#2439] [Ideas]
dear all

pls. implement an additional button in the lower left button bar to 'show/hide Celestial Sphere' that has the same effect as the tick in the respective configuration window (see below).

- thanks & best regards, mf
------------------------------------------------------------------
![Capture](https://user-images.githubusercontent.com/105195274/167355918-73284447-c6c9-4fad-ad39-c86ce1d97afe.JPG)

**Comment by gzotti** [comment]
We have a configurable hotkey action for this. Default: Ctrl+K, but you can reconfigure it.
